docs/extensions/tutorials/_posts/1970-01-10-DebugSettingsPages.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/debug-settings-pages
+title: Debugging and local development of settings pages
+section: Tutorials
+---
+
+# Debugging and Local Development
+<hr />
+
+This short tutorial covers how to improve your settings page development flow by utilizing the `Debug` feature in the Builder. It'll explain how to see code changes without having to `push` every code change and how to use dev tools to debug the settings pages through the Builder.
+
+> #### Note
+> HTML settings pages currently cannot use the `Debug` feature in the Builder.
+
+## Local development
+
+When you start developing settings pages, you'll quickly realize how much of a hassle it is to `shoutem publish` every change to see the updates on the Builder. To avoid this and enjoy a much more efficient development flow, you can simply do the following:
+
+1. Push, and install your extension in Shoutem Builder application (Only once)
+2. Locate to the `extName/server` directory and execute `npm run dev` to run the webpack dev server (Until you kill server there is no need to run it again)
```

Comment:
This is too much to remember. This is how I'd like it to work:
1. click debug icon in builder
A modal is shown instructing you how to serve your settings page locally by instructing you to execute a command in shoutem CLI positioned inside of extension directory. The command should display something like: `shoutem page debug <page-name>`, CLI would then run `npm run dev` inside of the appropriate server folder. Command should be copy/pasteable with copy to clipboard button. It should also display a link to this page, where security setup is explained.

2. paste the command in shoutem CLI positioned at extension folder, it runs dev env

3. develop :)

Reply:
@vedrani @ikatun what do you think? 🔝